src/src/PluginStructs/P129_data_struct.h

```diff
@@ -78,10 +78,10 @@ struct P129_data_struct : public PluginTaskData_base {
 
 private:
 
-  const uint32_t getChannelState(uint8_t offset,
-                                 uint8_t size) const;
+  uint32_t getChannelState(uint8_t offset,
+                           uint8_t size);
 
-  const bool     validChannel(uint channel) const {
+  bool     validChannel(uint channel) {
```

Comment:
Ditto, the function itself does not change the members, thus the function should be const.